test/extended/util/image_helpers.go

```diff
@@ -30,6 +30,21 @@ func ResetImage(tags map[string]string) {
 
 }
 
+//DumpImage is a helper that inspects the image along with some ginkgo debug
+func DumpImage(name string) {
+	g.By(fmt.Sprintf("Calling docker inspect for image %s", name))
```

Comment:
I would think that e2e.Logf would be more appropriate here.
g.By is used to describe test cases